HW03 Maria Evers.Rmd

```diff
@@ -0,0 +1,128 @@
+    sheet = "data for R", col_types = c("numeric", 
+        "numeric", "numeric", "numeric", 
+        "numeric"))
+
+#Previewing the structure of my dataset
+
+str(ATP)
+
+#Plotting standard graph with ggplot2
+
+ggplot(ATP, aes(x = factor(ATP_concentration), y = normalized_average)) +
+  geom_col(width = .5, color = "black", fill = "black") +
+  
+#Adding (visible) Error Bars to the graph 
+
+  geom_errorbar(aes(ymin = normalized_average - normalized_stdev, ymax = normalized_average + normalized_stdev), width = .2, color = "darkgray") +
+  
+#Adding axis labels 
+
+  labs(title = "Extracellular ATP internalization by H1299 NSCLC cells", x = "ATP concentration (uM)", y = "Relative intracellular ATP concentrations \n(control = 100%)") +
+  
+#Making the graph look nice 
+  
+  theme(plot.title = element_text(hjust = 0.5, size = 14), axis.text = element_text(size = 12, color = "black"), axis.title = element_text(size = 14), panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line.x.bottom = element_line(color = "black"), axis.line.y.left = element_line(color = "black")) +
+  
+#Adding stars for significance (pvalue calculated in Excel). I added each manually, but I am sure there are other ways to add these using pvalues as a column in your data. 
+  
+  annotate("text", label = "*" , x = 4, y = 150, color = "black", size = 7) +
+  annotate("text", label = "*" , x = 5, y = 150, color = "black", size = 7) +
+  annotate("text", label = "***" , x = 6, y = 175, color = "black", size = 7) +
+  annotate("text", label = "**" , x = 7, y = 175, color = "black", size = 7) +
+  annotate("text", label = "**" , x = 8, y = 185, color = "black", size = 7) +
+  annotate("text", label = "***" , x = 9, y = 210, color = "black", size = 7) +
+  annotate("text", label = "***" , x = 10, y = 275, color = "black", size = 7) 
+
+
+
+```
+
+### Next, I am graphing some RNASeq Data that we would like to highlight in an upcoming publication. 
+### This data is looking at changes in A549 NSCLC cells RNA expression levels in specific EMT-related genes after treatment with ATP or TGF-B for 2 or 6 hours.
+### I think this plot needs work, but I need to produce multi-panel figures like this with RNASeq data and Metabolomics data for a publication I'm working on, so **any input on this graph specifically is greatly appreciated!**
+
+```{r}
+library(readxl)
+
+#Loading in the Excel file. I went ahead and changed my data from the "wide" format to the "long" format in excel before importing. Instead of each treatment being its own column, treatment is now a column itself.
+
+RNASeqEMT <- read_excel("~/Documents/RNA seq analysis/Diff Exp Genes accumulated data.xlsx", 
+    sheet = "R data")
+    
+#Previewing structure of the data set 
+
+str(RNASeqEMT)
+
+#Graphing the dataset into a multi-panel bar chart, unfortunately I could not reorder my treatments to the desired order (control first) 
```

Comment:
I see, I did a quick Google search, try this? 
"# To change plot order of facet wrap,
# change the order of varible levels with factor()
diamonds$color <- factor(diamonds$color, levels = c("G", "J", "D", "E", "I", "F", "H"))
# Repeat first example with new order
d <- ggplot(diamonds, aes(carat, price, fill = ..density..)) +
xlim(0, 2) + stat_binhex(na.rm = TRUE) + theme(aspect.ratio = 1)
d + facet_wrap(~ color)"

https://www.rdocumentation.org/packages/ggplot2/versions/1.0.0/topics/facet_wrap